src/pages/community/index.page.tsx

```diff
@@ -12,7 +12,7 @@ import styles from './index.module.scss'
 
 import { CommunityHubFloatIconGroup } from './icons'
 
-const title = <div>Nervos Community Hub.</div>
+const title = <div>Nervos community hub.</div>
```

Comment:
The title is capitalized in https://docs.google.com/document/d/1abU8b1RSSDIXN3IquKbdT7ukxLCoIyaZg8F32-XtJow/edit

Reply:
This is the designer's comment https://nervos-official-website-git-preview-magickbase.vercel.app/community#thread-id=3NxPi